<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>characterTab</class>
 <widget class="QWidget" name="characterTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>813</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame { border-image: url(:/charsheet.png); }</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="prevCharButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>28</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/btn_left.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextCharButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>28</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/btn_right.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="spdEdit">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>210</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="consEdit">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>189</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="intEdit">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>170</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="dexEdit">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>150</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="strEdit">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>130</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="perEdit">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>230</y>
      <width>34</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>29</y>
      <width>375</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(255, 255, 255);
border: none</string>
    </property>
    <property name="text">
     <string>Roderick</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="inventoryScrollArea">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>548</y>
      <width>531</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
background-color: rgb(0, 0, 0);
}

QLabel {
background-color: rgb(255, 255, 255);
}

QScrollBar:vertical {
	background-color: rgb(0, 0, 0);
    width: 20px;
}

QScrollBar::up-arrow:vertical {
	border-image: url(:/up-arrow.png);
}
QScrollBar::down-arrow:vertical {
	border-image: url(:/down-arrow.png);
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="inventoryContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>509</width>
       <height>259</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
